src/style.css

```
@@ -1 +1,30 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
```

```suggestion
.stars__star:last-child {
  margin-right: 0;
}
```
You should disable margin-right for last child because you exactly this property for each star